composer.json

```diff
@@ -9,7 +9,10 @@
     }
   ],
   "require": {
-    "php": ">=7.1"
+    "php": ">=7.1",
+    "fond-of-spryker/thirty-five-up": "^dev-master",
+    "spryker/api": "^0.3.0",
+    "spryker/api-query-builder": "^0.1.0"
```

Comment:
here too

Reply:
spryker/api soll ich hier auch noch Version ab 1 erlauben oder was meinst du?
api-query-builder ist jede Version erlaubt

Reply:
`0.1.0 || 0.2.0 || 0.3.0`

Reply:
Macht das Sinn Versionen anzubieten wo man nicht gewährleisten kann das es funktioniert, wenn man die selber nicht nutzt?

Reply:
In dem Package hat sich nicht so viel geändert nur deprecations und improvements, deswegen haben wir gesagt wir können davon ausgehen dass es auch mit den alten Versionen einfach funktioniert. 

Hatte auch zuerst bedenken, aber in dem Fall wäre das in Ordnung.

Reply:
Ok, wie ihr wollt

Reply:
`"spryker/api": "^0.1.0 || ^0.2.0 || ^0.3.0",` for bugfix versions

Reply:
use `"spryker/api": "^0.1.0 || ^0.2.0 || ^0.3.0",` instead of `"spryker/api": "0.1.0 || 0.2.0 || 0.3.0",`